dev-python/sqlglot/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>parona@protonmail.com</email>
+		<name>Alfred Wingate</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
```

Comment:
Please add `python@` as co-maint. since this has working tests now (or at least I suppose they are working ;-)).

Reply:
Should mycli also be added as  its a pure python packages as well? Or is the policy that it being a different category disqualifies  it from being under the python project umbrella?

Reply:
Also should stabilise-allarches be in mycli as well. It applies equally here to my understanding

Reply:
mycli is not really a "generic" Python package whereas (at least to my understanding, please correct me if I'm wrong) `sqlglot` is a Python package providing modules used by other stuff.

Reply:
mycli appears to a normal python package, with it installing a python module and then having an entry point to it.

But it fails on the fact that its not intended to be used by other python packages. Which I guess is the sticking point. As checking yt-dlp, patool and so on, I see that none of them are under the python project umbrella.